perses/annihilation/new_relative.py

```diff
@@ -1305,8 +1305,8 @@ def handle_nonbonded(self):
 
                 # Add particle to the regular nonbonded force, but Lennard-Jones will be handled by CustomNonbondedForce
                 particle_index = self._hybrid_system_forces['standard_nonbonded_force'].addParticle(charge, sigma, 0.0)
-                # Charge will be turned on at lambda_electrostatics_delete = 0, off at lambda_electrostatics_delete = 1
-                self._hybrid_system_forces['standard_nonbonded_force'].addParticleParameterOffset('lambda_electrostatics_delete', particle_index, -charge, 0, 0)
+                # Charge will be turned off at lambda_electrostatics_delete = 0, on at lambda_electrostatics_delete = 1
+                self._hybrid_system_forces['standard_nonbonded_force'].addParticleParameterOffset('lambda_electrostatics_delete', particle_index, +charge, 0, 0)
```

Comment:
This sign change looks like has introduced a bug. If you want to change the way you define `lambda_electrostatics_delete` so that `0` is uncharged and `1` restores the old charges, you also need to change
```
                particle_index = self._hybrid_system_forces['standard_nonbonded_force'].addParticle(charge, sigma, 0.0)
```
to
```
                particle_index = self._hybrid_system_forces['standard_nonbonded_force'].addParticle(0.0, sigma, 0.0)
```